apps/api/src/app/app.module.ts

```diff
@@ -315,7 +311,6 @@ const autoSchemaFile = environment.production
         PassportsClientConfig,
         FileStorageConfig,
         FiskistofaClientConfig,
-        PowerBiConfig,
```

Comment:
Let's continue loading the PowerBiConfig here to make sure it's valid when the api starts. (Otherwise it can be silently ignored because of registerOptional)

Reply:
Done and done (regarding the first point, no I must've been too tunnel visioned and forgotten the write the @Inject(PowerBiConfig.KEY) inside the constructor param 😅